packages/valory/skills/dynamic_nft_abci/handlers.py

```diff
@@ -66,6 +66,7 @@
 OK_CODE = 200
 NOT_FOUND_CODE = 404
 BAD_REQUEST_CODE = 400
+AVERAGE_PERIOD_SECONDS = 10
```

Comment:
I've run +100 periods to get this value. This number represents how much it takes for the agents to complete a full period (not accounting for the reset and pause time), and it is rounded up a bit.